torch/distributions/exponential.py

```diff
@@ -39,3 +41,18 @@ def log_prob(self, value):
 
     def entropy(self):
         return 1.0 - torch.log(self.rate)
+
+    def natural_params(self):
+        return self._natural_params
+
+    @lazy_property
+    def _natural_params(self):
+        if isinstance(self.rate, Variable):
```

Comment:
Everything will be a variable soon. Once you merge in the latest master I believe you can assume `isinstance(-, Variable)` in all your implementations.